spec/datadog/tracing/distributed/trace_context_spec.rb

```diff
@@ -242,7 +242,7 @@
             let(:upstream_tracestate) { 'dd=old_value,other=vendor,dd=oops_forgot_to_remove_this' }
 
             it 'removes existing `dd=` values, prepending new `dd=` value' do
-              expect(tracestate).to eq('dd=o:origin,other=vendor')
+              expect(tracestate).to eq('dd=o:origin;future=field,other=vendor')
             end
```

Comment:
I think you either forgot to update the `upstream_tracestate` or you changed the wrong `expect` here. (Tests are breaking)

Reply:
Thank you!